src/OrchardCore.Modules/OrchardCore.Workflows/Views/WorkflowType/Index.cshtml

```diff
@@ -117,19 +118,27 @@
         $(".dropdown-menu .dropdown-item").filter(function() {
             return $(this).data("action");
         }).on("click", function () {
-            if ($(":checkbox[name='itemIds']:checked").length > 1) {
+            if ($(":checkbox[name='itemIds']:checked").length > 0) {
```

Comment:
```javascript
if ($(":checkbox[name='itemIds']").is(":checked")) {
```

Normally this should work too.

Reply:
I suggest to change it to what @Skrypt mentioned, while you are already did the almost of the work

Thanks

Reply:
I think this one was intentionally requiring 2 elements to be clicked so don't change it. If 2 elements are clicked then you allow doing bulk actions.